libs/wingsdk/src/std/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { Duration } from "./duration";
+
+export class Utils {
```

Comment:
Let's move this from `std` to `util`. I think it makes more sense to use it like this:

```js
bring util:

util.sleep(...);
```

And until we support globals:

```js
bring util;

util.Util.sleep();
```

Then we can change compiler import logic to treat classes with the same name as the namespace as "global".

Reply:
Sounds good, will do

Reply:
@eladb Let me know if you want me to open an issue for the compiler import logic